tests/congruence_tests/test_query.py

```diff
@@ -0,0 +1,1094 @@
+            query=models.RecommendQuery(
+                recommend=models.RecommendInput(
+                    positive=[10],
+                )
+            ),
+            with_payload=True,
+            limit=10,
+            using="image",
+        )
+
+    @classmethod
+    def dense_many_recommend(cls, client: QdrantBase) -> models.QueryResponse:
+        return client.query_points(
+            collection_name=COLLECTION_NAME,
+            query=models.RecommendQuery(
+                recommend=models.RecommendInput(
+                    positive=[10, 19],
+                )
+            ),
+            with_payload=True,
+            limit=10,
+            using="image",
+        )
+
+    @classmethod
+    def dense_discovery_image(cls, client: QdrantBase) -> models.QueryResponse:
+        return client.query_points(
+            collection_name=COLLECTION_NAME,
+            query=models.DiscoverQuery(
+                discover=models.DiscoverInput(
+                    target=10,
+                    context=models.ContextPair(positive=11, negative=19),
+                )
+            ),
+            with_payload=True,
+            limit=10,
+            using="image",
+        )
+
+    @classmethod
+    def dense_many_discover(cls, client: QdrantBase) -> models.QueryResponse:
+        return client.query_points(
+            collection_name=COLLECTION_NAME,
+            query=models.DiscoverQuery(
+                discover=models.DiscoverInput(
+                    target=10,
+                    context=[
+                        models.ContextPair(positive=11, negative=19),
+                        models.ContextPair(positive=12, negative=20),
+                    ],
+                )
+            ),
+            with_payload=True,
+            limit=10,
+            using="image",
+        )
+
+    @classmethod
+    def dense_context_image(cls, client: QdrantBase, limit: int) -> models.QueryResponse:
+        return client.query_points(
+            collection_name=COLLECTION_NAME,
+            query=models.ContextQuery(context=models.ContextPair(positive=11, negative=19)),
+            with_payload=True,
+            limit=limit,
+            using="image",
+        )
+
+    def dense_query_lookup_from(
+        self, client: QdrantBase, lookup_from: models.LookupLocation
+    ) -> models.QueryResponse:
+        return client.query_points(
+            collection_name=COLLECTION_NAME,
+            query=models.RecommendQuery(
+                recommend=models.RecommendInput(positive=[1, 2], negative=[3, 4])
+            ),
+            using="text",
+            limit=10,
+            lookup_from=lookup_from,
+        )
+
+    @classmethod
+    def no_query_no_prefetch(cls, client: QdrantBase) -> models.QueryResponse:
+        return client.query_points(collection_name=COLLECTION_NAME, limit=10)
+
+
+# ---- TESTS  ---- #
+
+
+def test_dense_query_lookup_from_another_collection():
+    fixture_points = generate_fixtures()
```

Comment:
```suggestion
    fixture_points = generate_fixtures(10)
```

Reply:
one of the reasons to introduce this test was to check that we don't exclude lookup collection point ids from the current collection, if we have a lot of points in the original collection, we might not be able to track this issue

Reply:
Good point